packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { registerDestructor } from '@ember/destroyable';
+
+interface HdsAppSideNavSignature {
```

Comment:
Should I export the signature? @alex-ju

Reply:
Yes, please, that will allow be to merge #2499 now

Reply:
Done